lib/media/media_source_capabilities.js

```diff
@@ -42,6 +42,24 @@ shaka.media.Capabilities = class {
         // eslint-disable-next-line no-restricted-syntax
         !!sourceBuffer.prototype && !!sourceBuffer.prototype.changeType;
   }
+
+  /**
+   * Determine support for MediaSource.setLiveSeekableRange and
+   * MediaSource.clearLiveSeekableRange
```

Comment:
This should probably explain why those API calls are required for "infinite live stream duration", since the relationship isn't immediately obvious. Something like...

```suggestion
   * MediaSource.clearLiveSeekableRange, which can allow for a media element
   * duration of Infinite by providing a non-infinite seekable range.
```